host/4/mariner/dotnet-isolated/dotnet7-isolated-mariner.Dockerfile

```diff
@@ -37,5 +38,6 @@ RUN dnf install -y glibc-devel
 
 COPY --from=runtime-image [ "/azure-functions-host", "/azure-functions-host" ]
 COPY --from=aspnet7 [ "/usr/share/dotnet", "/usr/share/dotnet" ]
+COPY --from=dn8-sdk-image [ "/usr/share/dotnet", "/usr/share/dotnet" ]
```

Comment:
Why do we need to include the dotnet8 sdk image

Reply:
without this running the docker image is failing. I merged the PR as the changes are close to green build. @CooperLink - I think its good for you take a final pass after I get all the green builds to see if there is any other fixes we need to do!